packages/components/src/tooltip/index.tsx

```diff
@@ -50,6 +50,7 @@ function UnforwardedTooltip(
 		position,
 		shortcut,
 		text,
+		className,
```

Comment:
Nit: The pre-existing props are alphabetically sorted, so perhaps we should move this one to the top to keep that order.